src/views/v2/rsvp/ari/form/template/fields.php

```diff
@@ -26,6 +26,10 @@
  */
 $has_meta = apply_filters( 'tribe_tickets_rsvp_has_meta', $rsvp, false );
 
+if ( $rsvp instanceof Tribe__Tickets__Ticket_Object ) {
```

Comment:
The same comment applies here.

Reply:
You're right this was meant for `$has_meta`

```suggestion
if ( $has_meta instanceof Tribe__Tickets__Ticket_Object ) {
```